web-portlet/src/java/org/sakaiproject/portlets/PortletIFrame.java

```diff
@@ -904,7 +912,18 @@ else if (newPageTitle.length() > MAX_TITLE_LENGTH)
             {
                 if ((infoUrl != null) && (infoUrl.length() > 0) && (!infoUrl.startsWith("/")) && (infoUrl.indexOf("://") == -1))
                 {
-                    infoUrl = "http://" + infoUrl;
+                    infoUrl = "//" + infoUrl;
```

Comment:
Although this seems good, I'm not 100% sure it's the right thing todo. I wonder if this was there so that if someone enters a URL of www.university.edu/course/STAT-101 it prepends the http:// onto it. However if the URL is for a different server there's no guarantee the server's protocol is any better than http.

What do you think?